src/main/java/core/basesyntax/AreaCalculation.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculation {
```

Comment:
Consider renaming the interface to `AreaCalculable` to better fit the convention of using adjectives for interface names. 😊